include/rangers-api/Application/Game/GameMode.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+namespace app::game {
+    class GameMode : public hh::fnd::Messenger {
+    public:
+        hh::game::GameManager* gameManager;
+        uint64_t unk2;
+        uint16_t unk3;
+        uint32_t unk4;
+        uint64_t fsm[9]; // tinyfsm
```

Comment:
Use the `INSERT_PADDING` macro instead of defining a `uint64_t` array and then add a comment. Also properly capitalize TinyFsm.